doc/APIGuide.md

```diff
@@ -19,21 +19,22 @@ The ethos of the OpenSim API is that the code is modular, reusable, and easily e
 
 ## Organization of OpenSim {#organization}
 
-OpenSim is built on the computational and simulation core provided by SimTK. This core includes low-level, efficient math and matrix algebra libraries, such as LAPACK, as well as the infrastructure for defining a dynamic System and its State. One can think of the System as the set of differential equations, and the state as its variables. 
+OpenSim is built on the computational and simulation libraries provided by SimTK. This core includes low-level, efficient math and matrix algebra libraries, such as LAPACK, as well as the infrastructure for defining a dynamic System and its State. One can think of the System as the set of differential equations, and the state as its variables. 
```

Comment:
Consider:

OpenSim is built on SimTK computational and simulation libraries that provide low-level, ...

Reply:
No comma after low-level either.